app/test_main.py

```diff
@@ -1,3 +1,33 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_zero_if_zero_input() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_should_return_zero_if_less_than_cat_year() -> None:
```

Comment:
This test case is similar to the previous one. Consider consolidating tests that cover the same logic to avoid repetition, as suggested by the checklist .